dial_opts.go

```diff
@@ -57,6 +57,22 @@ func WithCustomCerts(certPath string, insecureSkipVerify bool) grpc.DialOption {
 	}))
 }
 
+// WithCustomCertBytes is a dial option for requiring TLS from a []byte.
+//
+// This function panics if custom certificate pool cannot be instantiated.
+func WithCustomCertBytes(cert []byte, insecureSkipVerify bool) grpc.DialOption {
+	certPool := x509.NewCertPool()
+	ok := certPool.AppendCertsFromPEM(cert)
+	if !ok {
+		panic("No x509 Certificates parsed")
```

Comment:
Ideally this actually returns the same error as this: https://github.com/authzed/grpcutil/blob/f813f77722e58f97bbbba0766d9b14b3308e7374/x509util.go#L37

I think there are going to be too many scenarios where users might specify the wrong file and this function should actually fail rather than panic.